src/elements/content-sidebar/MetadataInstanceEditor.tsx

```diff
@@ -22,9 +24,8 @@ export interface MetadataInstanceEditorProps {
     template: MetadataTemplateInstance;
 }
 
-const MetadataInstanceEditor: React.FC<MetadataInstanceEditorProps> = ({
+export const MetadataInstanceEditor: React.FC<MetadataInstanceEditorProps> = ({
```

Comment:
nit: if we're exporting for unit tests, we usually add the export at bottom with something like:
```
export { MetadataInstanceEditor as MetadataInstanceEditorComponent };
```
example:
https://github.com/box/box-ui-elements/blob/b6b61bf47429541b1cbb1091fd94024c4ceeb6c8/src/elements/content-sidebar/Sidebar.js#L342-L343

Reply:
We actually don't need the `withApiWrapper` so I will restore the old way we were exporting here.